Ivanova/Menu.js

```diff
@@ -0,0 +1,176 @@
+function Meal() {
+    this.price = 0;
```

Comment:
All your hamburgers will have properties "price" and "calories" with value "0". You may pass them into Meal function and set.
![image](https://user-images.githubusercontent.com/31970322/52565927-477dd600-2e19-11e9-8e01-c508c94898f6.png)
The same situation with Salad and Drink